fe/fe-core/src/main/java/org/apache/doris/transaction/DatabaseTransactionMgr.java

```diff
@@ -941,7 +941,20 @@ public void finishTransaction(long transactionId, Set<Long> errorReplicaIds) thr
                                 }
                             }
 
-                            if (healthReplicaNum < quorumReplicaNum) {
+                            if (transactionState.isPublishTimeout() && healthReplicaNum > 0) {
```

Comment:
I suggest to add a config to enable this feature, and default is disabled.
Because the clone speed may not catch up the load speed, causing data with single replica for a long time.